tools/task_plots/swift_hardcoded_data.py

```diff
@@ -129,6 +129,13 @@
     "sink_do_gas_swallow",
     "rt_gradient",
     "rt_transport",
```

Comment:
this list doesn't match what's present in `src/task.h`.

We're missing `task_subtype_grav_counts` after `sf_counts`.

But it may be better to fix that in the upstream branch first and then pull those fixes in later.